packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -122,16 +123,21 @@ class MinimumTapTargetGuideline extends AccessibilityGuideline {
 
   @override
   FutureOr<Evaluation> evaluate(WidgetTester tester) {
-    return _traverse(
-      tester,
-      tester.binding.pipelineOwner.semanticsOwner!.rootSemanticsNode!,
-    );
+    Evaluation result = const Evaluation.pass();
+    for (final FlutterView view in tester.platformDispatcher.views) {
+      result += _traverse(
+        view,
+        tester.binding.pipelineOwner.semanticsOwner!.rootSemanticsNode!,
```

Comment:
nit: maybe add the same TODO about the root semanticsNode here?

Reply:
Done.